src/atlas/meshgenerator/detail/StructuredMeshGenerator.cc

```diff
@@ -142,6 +138,16 @@ StructuredMeshGenerator::StructuredMeshGenerator( const eckit::Parametrisation&
         }
         options.set( "partitioner", partitioner );
     }
+     else if ( grid::Partitioner::exists( "trans" ) ) {
+        partitioner = "trans";
+    } else {
+        partitioner = "equal_regions";
+    }
+    std::cout << "StructuredMeshGenerator::partitioner = " << partitioner << std::endl;
+    options.set( "partitioner", partitioner );
```

Comment:
Given that this statement is now here, the one at (new) line 139 can be removed.

Reply:
Done!

Reply:
Done